src/DateTime.php

```diff
@@ -3,11 +3,17 @@
 namespace Contributte\Utils;
 
 use DateTimeInterface;
+use DateTimeZone;
 use Nette\Utils\DateTime as NetteDateTime;
 
 class DateTime extends NetteDateTime
 {
 
+	public function __construct(string $time = 'now', ?DateTimeZone $timezone = null)
+	{
+		parent::__construct($time, $timezone);
```

Comment:
Because of nette/di: is_a() expects parameter 2 to be string, null given

Reply:
Well, I don't get it. :-) What kind of problem does it solve?

Reply:
It's solved in nette/di (https://github.com/nette/di/issues/186). Does it still required?

Reply:
Yes, the only thing changed is error message to "Service 'xxx' (type of Contributte\Utils\IDateTimeFactory): Type hint for $time in create() doesn't match type hint in Contributte\Utils\DateTime constructor"

Reply:
I see. Thanks.